godot/src/logic/player/camera_mode_area_detector.tscn

```diff
@@ -8,8 +8,8 @@ radius = 0.25
 height = 1.5
 
 [node name="camera_mode_area_detector" type="Area3D"]
-collision_layer = 256
-collision_mask = 256
+collision_layer = 2147483648
```

Comment:
Is it okay modying the camera_mode_area ?

Reply:
Yes, this was wrong.